src/utils/logging/log_stdout.rs

```diff
@@ -0,0 +1,74 @@
+use log::{Level, Log, Metadata, Record};
+use std::io::{stdout, Stdout, Write};
+use std::sync::Mutex;
+
+use colored::*;
+
+use crate::utils::time::time_since_launched;
+
+pub struct LogStdOut {
+    stdout: Mutex<Stdout>,
```

Comment:
would this remove the need for the mutex ? `https://doc.rust-lang.org/std/io/struct.StdoutLock.html`